ideascube/utils.py

```diff
@@ -6,3 +6,13 @@ class classproperty(property):
     """
     def __get__(self, cls, owner):
         return self.fget.__get__(None, owner)()
+
+
+# We do not use functool.partial cause we want to mock stderr for unittest
+# If we use partial we keep a ref to the original sys.stderr and output is not
+# captured.
+def printerr(*args, **kwargs):
+    import sys
```

Comment:
To not pollute the namespace.
This seems to me "contained enough" to have the import inside the function.
(And in a certain way, utils.py is not intended to import sys (even if it does *if we use printerr*))

But we can move it to top and (eventually) use __all__